src/actions/__test__/getArticlesAction.test.js

```diff
@@ -31,7 +31,7 @@ describe("actions", () => {
     return store.dispatch(getArticles()).then(() => {
       const dispathedActions = store.getActions();
 
-      expect(dispathedActions).toContainEqual({"isLoading": true, "type": "GET_ARTICLES_REQUEST"});
+      expect(dispathedActions).toContainEqual({ isLoading: true, type: "GET_ARTICLES_REQUEST" });
```

Comment:
Format this with prettier it seems you have exceeded the amount of characters for each line of code which is 80-85